test/test_common/tls_utility.cc

```diff
@@ -2,6 +2,7 @@
 
 #include "common/common/assert.h"
 
+#include "gtest/gtest.h"
```

Comment:
this utility is being used by `tls_inspector_benchmark` which is not linked to google test lib. that is the cause of CI failure.

Reply:
i am moving this function to a separate file in the test/common/ssl/test_utility.h so that this utility is not touched..